merqube_client_lib/templates/equity_baskets/util.py

```diff
@@ -59,12 +59,8 @@ def get_index_info(config: dict[str, Any], model: Type[ClientIndexConfigBase]) -
         if hc.swaps_monitor_codes:
             kwargs["swaps_monitor_codes"] = hc.swaps_monitor_codes
 
-    if kwargs == {}:
-        logger.warning("Warning, no calendar identifiers or swaps monitor codes found, defaulting to nyse")
-        kwargs["calendar_identifiers"] = ["MIC:XNYS"]
```

Comment:
ive determined this made no sense, especially since the first major usage of this is all european. i will put this in the template as the default, but if the user removes it it will be M_F